### PR TITLE
interactive: preload initial example(6/7)

### PR DESCRIPTION
**Pull request by dshaaban01**
preload initial example into gui so users can gain a quicker understanding + theme changes to make sure that text area is visible to everyone (@tobiasgrosser had some issues seeing the text in the text area as the background colour+text colour of TextArea's were similar ). 

Light Mode: 
image

Dark Mode:
image

### CODE REVIEW

**Review by compor — APPROVED** [review]
LGTM, an external config file for the preloaded text, the themes, etc might be something to consider in the future.

File: xdsl/interactive/app.py [hunk 1]
```diff
@@ -140,12 +131,8 @@ def compose(self) -> ComposeResult:
 
     def on_mount(self) -> None:
         """Configure widgets in this application before it is first shown."""
-
-        # register's the theme for the Input/Output TextArea's
-        self.input_text_area.register_theme(InputApp._DEFAULT_THEME)
-        self.output_text_area.register_theme(InputApp._DEFAULT_THEME)
-        self.input_text_area.theme = "my_theme_design"
-        self.output_text_area.theme = "my_theme_design"
+        self.input_text_area.theme = "vscode_dark"
```
Comment (superlopuh) [inline xdsl/interactive/app.py:134]: This is unrelated to the preloading of the text, could you please split out the two PRs? They also don't seem like they would need to be in a stack as they don't functionally depend on anything not in main

**Review by superlopuh — CHANGES_REQUESTED** [review]

File: xdsl/interactive/app.py (hunk 1 above)
Comment (superlopuh) [reply xdsl/interactive/app.py:134]: Yes

File: xdsl/interactive/app.py [hunk 2]
```diff
@@ -161,6 +164,19 @@ def on_mount(self) -> None:
         # initialize gui to inital state - uncondensed mode.
         self.condense_mode = False
 
+        # initialize GUI with an interesting input IR and pass application
+        self.input_text_area.load_text(
+            """func.func @hello(%n : index) -> index {
```
Comment (superlopuh) [inline xdsl/interactive/app.py:169]: Can you please move this out to an UPPER_CAMEL_CASE constant at the level of the file? Maybe called `INITIAL_IR_TEXT`, or something like that?

**Review by superlopuh — CHANGES_REQUESTED** [review]

**Review by webmiche — APPROVED** [review]

**Pull request closed without merging**